modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -42,6 +42,15 @@ public class UpgradeCatchAllCheck {
 			companyId, 551, "test", start, end);
 	}
 
+	public void method(
+		InfoItemReference infoItemReference,
+		LayoutDisplayPageProviderRegistry layoutDisplayPageProviderRegistry) {
+
+		LayoutDisplayPageProvider<?> layoutDisplayPageProvider =
+			layoutDisplayPageProviderRegistry.getLayoutDisplayPageProvider(
+				infoItemReference.getClassName());
+	}
```

Comment:
@alberttgomes is necessary these parameters and this layoutDisplayPageProvider declaration? If not, let's remove it and follow the pattern.

Reply:
I did this based on the test's check that already exists, I can refactor more. @nicolas-sss